### PR TITLE
[Fix-16517][Alert Instance] Alert plugin instance i18n wrong result in rendering frontend ui

### LINKED ISSUE #16517: [Bug] [Alert Instance] Alert plugin instance i18n wrong result in rendering frontend ui
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

Alert plugin instance i18n wrong result in rendering frontend ui. The language type of the page is abnormal in ui.

### What you expected to happen

Normal.

### How to reproduce

aboved.

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by sdhzwc**
## Purpose of the pull request
The repaired effect is as follows:
image
image

close #16517

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-api/src/main/java/org/apache/dolphinscheduler/alert/api/AlertInputTips.java [hunk 1]
```diff
@@ -73,4 +75,11 @@ public String getMsg() {
             return this.enMsg;
         }
     }
+
+    public static Map<String, String> getAllMsg(AlertInputTips alertInputTips) {
+        HashMap<String, String> allMsgMap = new HashMap<>(16);
```
Comment (SbloodyS) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-api/src/main/java/org/apache/dolphinscheduler/alert/api/AlertInputTips.java]:
```suggestion
        HashMap<String, String> allMsgMap = new HashMap<>();
```

**Review by SbloodyS — COMMENTED** [review]
Please add some tests for this change.

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-api/src/main/java/org/apache/dolphinscheduler/alert/api/AlertInputTips.java (hunk 1 above)
Comment (sdhzwc) [reply dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-api/src/main/java/org/apache/dolphinscheduler/alert/api/AlertInputTips.java]: > 

done

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/UiPluginServiceTest.java [hunk 2]
```diff
@@ -82,6 +86,12 @@ public void testQueryPluginDetailById() {
         Mockito.when(pluginDefineMapper.queryDetailById(1)).thenReturn(pluginDefine);
         result = uiPluginService.queryUiPluginDetailById(1);
         Assertions.assertEquals(Status.SUCCESS, result.get("status"));
+
+        PluginDefine data = (PluginDefine) result.get("data");
+        String pluginParams = data.getPluginParams();
+        ArrayNode arrayNode = JSONUtils.parseArray(pluginParams);
+        String placeholder = arrayNode.path(0).path("props").path("placeholder").asText();
+        Assertions.assertEquals(2, JSONUtils.toMap(placeholder).size());
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/UiPluginServiceTest.java]: We need to verify both `en` and `zh` in test.

Comment (sdhzwc) [reply dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/UiPluginServiceTest.java]: > We need to verify both `en` 和 `zh` in test.

done

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by ruanwenjun — COMMENTED** [review]
This is not an implementation I can accept, we already have two kinds of i18n implementation, right now we introduced another new one. We need to make sure for one plugin there only exist one i18n implementation. The i18n of parameter type  is controlled by front-end, place holder is controlled by front-end/back-end, this look strange. 

It will be great if the parameter type is also controlled at this way.

**Review by ruanwenjun — APPROVED** [review]

**Comment by sdhzwc** [comment]
> This is not an implementation I can accept, we already have two kinds of i18n implementation, right now we introduced another new one. We need to make sure for one plugin there only exist one i18n implementation. The i18n of parameter type is controlled by front-end, place holder is controlled by front-end/back-end, this look strange.
> 
> It will be great if the parameter type is also controlled at this way.

This time I am solving the display problem, and next I plan to extract all prompt information and put it on the front end, then retrieve the prompt information through specifying keys

**Comment by ruanwenjun** [comment]
> > This is not an implementation I can accept, we already have two kinds of i18n implementation, right now we introduced another new one. We need to make sure for one plugin there only exist one i18n implementation. The i18n of parameter type is controlled by front-end, place holder is controlled by front-end/back-end, this look strange.
> > It will be great if the parameter type is also controlled at this way.
> 
> This time I am solving the display problem, and next I plan to extract all prompt information and put it on the front end, then retrieve the prompt information through specifying keys

Glad to hear that you have plan, you can submit issue to track this.

**Pull request merged**